docs/src/main/sphinx/release/release-476.md

```diff
@@ -0,0 +1,195 @@
+# Release 476 (2 Jun 2025)
+
+## General
+
+* Add support for comparing values of `geometry` type. ({issue}`25225`)
+* {{breaking}} Require JDK 24 to run Trino. ({issue}`23498`)
+* Allow configuring `query.max-memory-per-node` and `memory.heap-headroom-per-node` 
+  relative to maximum heap size. ({issue}`25843`)
+* Add feature to deactivate the automated database schema migration for the database 
+  backend for resource groups with the property `resource-groups.db-migrations-enabled`. ({issue)`25451`)
+* Remove the [](/develop/example-http) from the tar.gz archive and the Docker container.  ({issue}`25128`)
+* Fix rare bug when server can hang under load. ({issue}`25816`)
+* Fix regression that prevented graceful shutdown from working. ({issue}`25690`)
+* Fix potential query failure when `fault_tolerant_execution_runtime_adaptive_partitioning_enabled` 
+  is set to `true`. ({issue}`25870`)
+* Fix failure for queries involving casts with `row` types. ({issue}`25864`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Docker image
+
+## CLI
+
+## BigQuery connector
+
+* {{breaking}} Require the `--sun-misc-unsafe-memory-access=allow` JVM configuration 
+  option to run Trino with the connector. ({issue}`25669`)
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+* Improve performance of selective joins for federated queries. ({issue}`25123`)
+
+## Delta Lake connector
+
+* Add support for the `FOR TIMESTAMP AS OF` clause. ({issue}`21024`)
+* Allow configuring *User-ManagedIdentity* in AzureFS. ({issue}`23447`)
+* Add signer type support to the native S3 filesystem. ({issue}`25820`)
+* Avoid skipping statistics computation for all columns when some column types 
+  don't support statistics. ({issue}`24487`)
+* Improve compatibility with S3-compliant storage systems. ({issue}`25791`)
+* Improve query planning performance. ({issue}`24570`)
+* Improve performance when reading tables. ({issue}`25826`)
+* Reduce S3 throttling failures. ({issue}`25781`)
+* Fix failure when reading `variant` type column after executing `optimize` 
+  procedure. ({issue}`25666`)
+* Fix query failures when attempting to read `date` columns stored as integer
+  values in Parquet files. ({issue}`25667`)
+* Fix `SHOW SCHEMAS` failure when catalog names contain mixed case letters. ({issue}`25701`)
```

Comment:
I think `SHOW SCHEMAS` was just an example of query which failed. The actual class of failed queres was larger.